src/main/java/com/williamfiset/algorithms/geometry/MinimumCostPolygonTriangulation.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * This file shows you how to find the minimum cost polygon triangulation of a set of points.
+ *
+ * <p>Time Complexity: O(n^3)
+ *
+ * @author Bryan Bowles
+ */
+package com.williamfiset.algorithms.geometry;
+
+import java.awt.geom.Point2D;
+
+// Problem explanation: https://www.geeksforgeeks.org/minimum-cost-polygon-triangulation/
+public class MinimumCostPolygonTriangulation {
```

Comment:
This code makes an assumption that the polygon is Convex and that the points are in CW and CCW order, correct? Can you either rename the class, add a comment, or both to reflect that?

Reply:
Good catch! updated